sbws/lib/v3bwfile.py

```diff
@@ -210,7 +190,11 @@ def num_lines(self):
 
     @staticmethod
     def generator_started_from_file(conf):
-        return read_started_ts(conf)
+        state = State(conf['paths']['state_fname'])
```

Comment:
I'd just add here a comment (or docstring) about what is supposed to be read here.
Something like "ISO formated timestamp which represents the date and time...."